src/components/LeftSideBar/LeftSidebar.js

```diff
@@ -16,68 +22,172 @@ import MoreIcon from "./Icons/MoreIcon";
+        Icon={ProfileIcon}
+        text="Profile"
+        handlerFunction={() => console.log("Navigate to profile")}
+      />
+      <SidebarOption
+        Icon={MoreIcon}
+        text="More"
+        handlerFunction={() => console.log("Navigate to more stuff")}
+      />
+      <button className={styles.tweetButton} onClick={handleTweetBoxOpen}>
+        Tweet
+      </button>
 
-	return (
-		<div className={styles.container}>
-			<TwitterIcon className={styles.twitterIcon} style={{ fontSize: 35 }} />
-			<SidebarOption
-				Icon={HomeIcon}
-				text="Home"
-				handlerFunction={() => console.log("Navigate to home")}
-			/>
-			<SidebarOption
-				Icon={ExploreIcon}
-				text="Explore"
-				handlerFunction={() => console.log("Navigate to explore")}
-			/>
-			<SidebarOption
-				Icon={NotificationsIcon}
-				text="Notifications"
-				handlerFunction={() => console.log("Navigate to notifications")}
-			/>
-			<SidebarOption
-				Icon={MessagesIcon}
-				text="Messages"
-				handlerFunction={() => console.log("Navigate to messages")}
-			/>
-			<SidebarOption
-				Icon={BookmarkIcon}
-				text="Bookmarks"
-				handlerFunction={() => console.log("Navigate to bookmarks")}
-			/>
-			<SidebarOption
-				Icon={ListIcon}
-				text="Lists"
-				handlerFunction={() => console.log("Navigate to lists")}
-			/>
-			<SidebarOption
-				Icon={ProfileIcon}
-				text="Profile"
-				handlerFunction={() => console.log("Navigate to profile")}
-			/>
-			<SidebarOption
-				Icon={MoreIcon}
-				text="More"
-				handlerFunction={() => console.log("Navigate to more stuff")}
-			/>
-			<button className={styles.tweetButton} onClick={handleOpen}>
-				Tweet
-			</button>
-			<Modal open={open} onClose={handleClose} className={styles.tweetModal}>
-				<div className={styles.tweetModalContent}>Tweet</div>
-			</Modal>
-		</div>
-	);
+      <div
+        className={styles.bottomProfileManagement}
+        aria-controls="simple-menu"
+        aria-haspopup="true"
+        onClick={(event) => {
+          handleMenuOpen(event);
+          console.log("hey");
+        }}
+      >
+        <AccountCircleIcon fontSize="large" style={{ color: "#1da1f2" }} />
+        <div className={styles.nameAndHandle}>
+          <div>Sam Louderback</div>
+          <div>@samlouderback10</div>
+        </div>
+        <SvgIcon className={styles.threeDots}>
+          <path d="M19.39 14.882c-1.58 0-2.862-1.283-2.862-2.86s1.283-2.862 2.86-2.862 2.862 1.283 2.862 2.86-1.284 2.862-2.86 2.862zm0-4.223c-.75 0-1.362.61-1.362 1.36s.61 1.36 1.36 1.36 1.362-.61 1.362-1.36-.61-1.36-1.36-1.36zM12 14.882c-1.578 0-2.86-1.283-2.86-2.86S10.42 9.158 12 9.158s2.86 1.282 2.86 2.86S13.578 14.88 12 14.88zm0-4.223c-.75 0-1.36.61-1.36 1.36s.61 1.362 1.36 1.362 1.36-.61 1.36-1.36-.61-1.363-1.36-1.363zm-7.39 4.223c-1.577 0-2.86-1.283-2.86-2.86S3.034 9.16 4.61 9.16s2.862 1.283 2.862 2.86-1.283 2.862-2.86 2.862zm0-4.223c-.75 0-1.36.61-1.36 1.36s.61 1.36 1.36 1.36 1.362-.61 1.362-1.36-.61-1.36-1.36-1.36z"></path>
```

Comment:
Is this Icon not used anywhere else? should we not make this a separate component?